### PR TITLE
Fix/general improvements

### PR DESCRIPTION
**Pull request by cremebrule**

### CODE REVIEW

File: omnigibson/scenes/interactive_traversable_scene.py [hunk 1]
```diff
@@ -138,10 +138,10 @@ def filter_rooms_and_object_categories(
                 load_room_instances = [load_room_instances]
             load_room_instances_filtered = []
             for room_instance in load_room_instances:
-                if room_instance in self._seg_map.room_ins_name_to_ins_id:
-                    load_room_instances_filtered.append(room_instance)
-                else:
+                load_room_instances_filtered.append(room_instance)
+                if room_instance not in self._seg_map.room_ins_name_to_ins_id:
                     log.warning("room_instance [{}] does not exist.".format(room_instance))
+                load_room_instances_filtered.append(room_instance)
```
Comment (cgokmen) [inline omnigibson/scenes/interactive_traversable_scene.py:144]: You're appending twice

File: omnigibson/robots/behavior_robot.py [hunk 2]
```diff
@@ -139,11 +140,11 @@ def usd_path(self):
     def model_name(self):
         return "BehaviorRobot"
 
-    @property
+    @classproperty
     def n_arms(self):
```
Comment (cgokmen) [inline omnigibson/robots/behavior_robot.py:144]: Should this still be `self` and not `cls` if its a class property? not sure.

File: omnigibson/simulator.py [hunk 3]
```diff
@@ -230,8 +230,8 @@ class Simulator(lazy.omni.isaac.core.simulation_context.SimulationContext, Seria
         def __init__(
             self,
             gravity=9.81,
-            physics_dt=1.0 / 120.0,
-            rendering_dt=1.0 / 30.0,
+            physics_dt=1.0 / gm.DEFAULT_PHYSICS_FREQ,
+            rendering_dt=1.0 / gm.DEFAULT_RENDERING_FREQ,
```
Comment (cgokmen) [inline omnigibson/simulator.py:234]: This is bad - it gets processed at import time so changing the macro does nothing. Can we default this into None and then do `if physics_dt is None: physics_dt = 1.0 / gm. ...` down the line

File: omnigibson/utils/ui_utils.py [hunk 4]
```diff
@@ -485,7 +485,7 @@ def get_interpolated_positions(step):
                 pan_angle = np.arctan2(-xy_direction[0], xy_direction[1])
                 tilt_angle = np.arcsin(z)
                 # Infer global quat orientation from these angles
-                quat = T.euler2quat([np.pi / 2 - tilt_angle, 0.0, pan_angle])
+                quat = T.euler2quat([np.pi / 2 + tilt_angle, 0.0, pan_angle])
```
Comment (cgokmen) [inline omnigibson/utils/ui_utils.py:488]: Assuming you have tested this

File: omnigibson/robots/behavior_robot.py (hunk 2 above)
Comment (cremebrule) [reply omnigibson/robots/behavior_robot.py]: fixed!

File: omnigibson/simulator.py (hunk 3 above)
Comment (cremebrule) [reply omnigibson/simulator.py]: fixed

**Review by ChengshuLi — APPROVED** [review]

**Pull request merged**